network.js

```diff
@@ -43,16 +43,15 @@ function setJavaProxyArgs(fields) {
     `-Dhttp.proxyHost=${fields.hostname}`,
     `-Dhttp.proxyPort=${fields.port}`,
     `-Dhttps.proxyHost=${fields.hostname}`,
-    `-Dhttps.proxyPort=${fields.port}`
+    `-Dhttps.proxyPort=${fields.port}`,
+    `-Dhttp.nonProxyHosts="localhost|127.0.0.1|0.0.0.0"`
```

Comment:
@ahmedalsudani Looking more closely now, I think we might be better off to leave the nonProxyHosts param out https://docs.oracle.com/javase/7/docs/api/java/net/doc-files/net-properties.html